tests/fixtures/kamereon/vehicle_data/hvac-settings-withschedule.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
I think it is enough to rename this file "hvac-settings.json" and remove the fixture with all inactive.